source/neuropod/bindings/java/src/main/native/utils.cc

```diff
@@ -46,6 +48,35 @@ jclass findClass(JNIEnv *env, const char *name)
     return ret;
 }
 
+jmethodID getMethodID(JNIEnv *env, jclass clazz, const char *name, const char *sig)
+{
+    jmethodID ret = env->GetMethodID(clazz, name, sig);
+    if (reinterpret_cast<jlong>(ret) == 0)
+    {
+        throw std::runtime_error(std::string("Method ID not found: ") + name + sig);
+    }
+    return ret;
+}
+
+jobject getFieldObject(JNIEnv *env, jclass dataTypes, std::string fieldName)
+{
+    jfieldID field = env->GetStaticFieldID(dataTypes, fieldName.c_str(), TENSOR_TYPE.c_str());
+    if (reinterpret_cast<jlong>(field) == 0)
+    {
+        throw std::runtime_error(std::string("Field not found: ") + fieldName);
+    }
+    return env->GetStaticObjectField(dataTypes, field);
+}
```

Comment:
Should we rename this function to be more clear? It seems specific to `TENSOR_TYPE`